test/UID2.Client.Test/BidstreamClientTests.cs

```diff
@@ -302,21 +302,52 @@ private void UserOptedOutTest(TokenVersion tokenVersion)
         }
 
         [Theory]
-        // These are the domains associated with site SITE_ID, as defined by KeySharingResponse();
+        // These are the domain or app names associated with site SITE_ID, as defined by KeySharingResponse();
         [InlineData("example.com", TokenVersion.V2)]
         [InlineData("example.org", TokenVersion.V2)]
+        [InlineData("com.123.Game.App.android", TokenVersion.V2)]
+        [InlineData("123456789", TokenVersion.V2)]
         [InlineData("example.com", TokenVersion.V3)]
         [InlineData("example.org", TokenVersion.V3)]
+        [InlineData("com.123.Game.App.android", TokenVersion.V3)]
+        [InlineData("123456789", TokenVersion.V3)]
         [InlineData("example.com", TokenVersion.V4)]
         [InlineData("example.org", TokenVersion.V4)]
-        private void TokenIsCstgDerivedTest(string domainName, TokenVersion tokenVersion)
+        [InlineData("com.123.Game.App.android", TokenVersion.V4)]
+        [InlineData("123456789", TokenVersion.V4)]
+        private void TokenIsCstgDerivedTest(string domainOrAppName, TokenVersion tokenVersion)
         {
             Refresh(KeySharingResponse(new[] { MASTER_KEY, SITE_KEY }));
 
             var privacyBits = PrivacyBitsBuilder.Builder().WithClientSideGenerated(true).Build();
             string advertisingToken = _tokenBuilder.WithPrivacyBits(privacyBits).WithVersion(tokenVersion).Build();
             ValidateAdvertisingToken(advertisingToken, IdentityScope.UID2, IdentityType.Email, tokenVersion);
-            var res = _client.DecryptTokenIntoRawUid(advertisingToken, domainName);
+            var res = _client.DecryptTokenIntoRawUid(advertisingToken, domainOrAppName);
+            Assert.True(res.IsClientSideGenerated);
+            Assert.True(res.Success);
+            Assert.Equal(DecryptionStatus.Success, res.Status);
+            Assert.Equal(EXAMPLE_EMAIL_RAW_UID2_V2, res.Uid);
+        }
+        
+        [Theory]
+        // These are the domain or app names associated with site SITE_ID but vary in capitalization, as defined by KeySharingResponse();
+        [InlineData("example.com", TokenVersion.V2)]
```

Comment:
are these tests cases really _vary in capitalization_? they look like exact same casing as defined in `KeySharingResponse()`?
i would suggest varying inlineData like:

Example.com
Example.org
com.123.game.app.android
etc...

so flipping the casing?

Reply:
My bad, I forget to change it. Flipped the casing.